lib/build_targets/package.dart

```diff
@@ -145,6 +145,7 @@ class DotnetTpk extends TizenPackage {
       '-o',
       '${outputDir.path}/', // The trailing '/' is needed.
       tizenProject.editableDirectory.path,
+      '/p:DefineConstants=${buildInfo.deviceProfile.toUpperCase()}_PROFILE',
```

Comment:
Hmm, I'm not sure which way is the standard but wouldn't it be better to make the option come before the argument (directory path)?

Reply:
There is no special reason. I was worried that it might affect other flags. I modified according to your comments.